development/custom-image/install-deps-debian.sh

```diff
@@ -39,6 +39,22 @@ sudo add-apt-repository \
 sudo apt update
 sudo apt install -y docker-ce=${DOCKER_VERSION}
 
+# setup logging drived for docker
+cat << EOF > /etc/docker/daemon.json
+{
+   "log-driver": "gcplogs",
+   "log-opts": {
+         "gcp-meta-name": "kyma-integration-test"
+      }
+}
+EOF
+sudo mv /tmp/daemon.json /etc/docker/
+
+# install fluentd 
+curl -Lo /tmp/install-logging-agent.sh https://dl.google.com/cloudagents/install-logging-agent.sh && \
```

Comment:
Is there a possibility to pinpoint this to a specific version? This seems to always install whatever is specified in the script and could possibly use a different version than we want it to.